### PR TITLE
docs: remove SSH and EC2 key pairs

### LINKED ISSUE #560: README still shows SSH as option for connecting to instance
SSH was removed but README still lists SSH and EC2 key pair as options for accessing the instance.

### PR DESCRIPTION
**Pull request by kayman-mk**
## Description

Removes outdated documentation from the module regarding SSH and EC2 key pair connection method.

Closes #560 

## Migrations required

No

## Verification

No verifications done, documentation changed only.

### CODE REVIEW

**Comment by kayman-mk** [comment]
@npalm Could you please check why we need this directory? https://github.com/npalm/terraform-aws-gitlab-runner/tree/develop/examples/runner-multi-region/_docs

May be we can integrate it into the docs one level above?

**Comment by npalm** [comment]
> @npalm Could you please check why we need this directory? https://github.com/npalm/terraform-aws-gitlab-runner/tree/develop/examples/runner-multi-region/_docs
> 
> May be we can integrate it into the docs one level above?

Will do!

**Comment by kayman-mk** [comment]
The docs I deleted seems to be outdated and a duplication of the module documentation one level above.

**Comment by kayman-mk** [comment]
@npalm PR is complete and ready now.

File: README.md [hunk 1]
```diff
@@ -159,8 +159,6 @@ persist that contains the packaged Lambda function.
 
 A few option are provided to access the runner instance:
 
-1.  Provide a public ssh key to access the runner by setting \`\`.
-2.  Provide a EC2 key pair to access the runner by setting \`\`.
 3.  Access via the Session Manager (SSM) by setting `enable_runner_ssm_access` to `true`. The policy to allow access via SSM is not very restrictive.
```
Comment (baolsen) [inline README.md:162]: Minor: Should maybe update the numbering here

**Comment by baolsen** [comment]
For what its worth, I had a look and this does address all the SSH references I found in #560

File: README.md (hunk 1 above)
Comment (kayman-mk) [reply README.md:162]: Just checked a linter. Yes, ordered lists should start with 1.

**Review by npalm — APPROVED** [review]

**Pull request merged**